### ISSUE TITLE
Work on quotes without spaces and nested quotes

### ISSUE THREAD

**Issue by YousufSSyed** [roodolv/markdown-toggle.nvim#4]
The following are valid markdown, but when I try to insert lines below any of the following, > characters aren't created.

>quote
>>nested quote
>>>double nested quote

```md
>quote
>>nested quote
>>>double nested quote
```

Is this plugin supposed to work with `o` and `O`?

**Comment by YousufSSyed** [comment]
>  The logic needs to be more precise to handle cases without a space.

Could you just detect any lines that start with any number of >s? 

Writing `>` in Obsidian without a space counts as a blockquote

**Comment by roodolv** [comment]
> Writing `>` in Obsidian without a space counts as a blockquote

Yes, I already understand how this plugin differs from Obsidian and how it should work.
Please be patient for a while.

**Comment by YousufSSyed** [comment]
ok

**Comment by lmdevv** [comment]
+1

**Comment by roodolv** [comment]
Sorry for the long wait. I've finally implemented this feature in PR #12.
It will be included in the next release, so please update the plugin if it's published.

By the way, this plugin toggles a quotation mark together with a whitespace by default.
Let me know if you'd prefer an option to toggle quotes without a whitespace.

**Comment by roodolv** [comment]
Now I've completely fixed this in the PR https://github.com/roodolv/markdown-toggle.nvim/pull/17.

This plugin now correctly detects successive quotation marks on the beginning of the line as follows:

**NOTE**: `|` is a cursor

```md
>>>hoge|

↓ `autolist()`

>>>hoge
>>>|
```

```md
> > > hoge|

↓ `autolist()`

> > > hoge
> > > |
```

In addition, now you can execute `quote()` on successive quotation marks:

```md
> > > fuga

↓ `quote()`

> > fuga
```

```md
>>>fuga

↓ `quote()`

>>fuga
```
